.github/workflows/linux_sanitizers.yml

```diff
@@ -468,3 +469,86 @@ jobs:
           name: test-results-cpp
           path: ${{ env.INSTALL_TEST_DIR }}/TEST*.xml
           if-no-files-found: 'error'
+  CPU_Functional_Tests:
+    name: CPU functional tests
+    timeout-minutes: 30
+    runs-on: ${{ inputs.runner }}
+    container:
+      image: ${{ inputs.image }}
+    defaults:
+      run:
+        shell: bash
+    env:
+      DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
+      INSTALL_DIR: ${{ github.workspace }}/install
+      INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
+      PARALLEL_TEST_SCRIPT: ${{ github.workspace }}/install/tests/functional_test_utils/layer_tests_summary/run_parallel.py
+      PARALLEL_TEST_CACHE: ${{ github.workspace }}/install/tests/test_cache.lst
+    steps:
+      - name: Download OpenVINO package
+        uses: actions/download-artifact@65a9edc5881444af0b9093a5e628f2fe47ea3b2e # v4.1.7
+        with:
+          name: openvino_package
+          path: ${{ env.INSTALL_DIR }}
+
+      - name: Download OpenVINO tests package
+        uses: actions/download-artifact@65a9edc5881444af0b9093a5e628f2fe47ea3b2e # v4.1.7
+        with:
+          name: openvino_tests
+          path: ${{ env.INSTALL_TEST_DIR }}
+
```

Comment:
Please take a look at how these steps are implemented in the C++ tests job: https://github.com/openvinotoolkit/openvino/blob/06fbf729a5e83332b978445adbb63115c600a5c4/.github/workflows/linux_sanitizers.yml#L233, especially the `name` args.